src/main/java/com/bookshop/controller/BookController.java

```diff
@@ -29,7 +36,19 @@ public BookDto getBookById(@PathVariable Long id) {
     }
 
     @PostMapping
-    public BookDto createBook(@RequestBody CreateBookRequestDto requestDto) {
+    @ResponseStatus(HttpStatus.CREATED)
+    public BookDto createBook(@RequestBody @Valid CreateBookRequestDto requestDto) {
         return bookService.save(requestDto);
     }
+
+    @DeleteMapping("/{id}")
+    public void deleteById(@PathVariable Long id) {

```

Comment:
```suggestion
    public void deleteById(@PathVariable @Positive Long id) {

```

Reply:
I have it in the branch Pagination so it will be added after a merge